[redesign] Timestamp page

thi4go opened the pull request:
Opening a wip so that fellow colleagues can check the state of the components. This PR introduces the timestamping page with the new design specs.

~~Build on top #89~~ _merged_
Closes #74 
Part of #85

tiagoalvesdulce commented on src/components/TimestampForm/timestampform.module.css:
```diff
@@ -0,0 +1,146 @@
+    display: flex;
+    z-index: 999;
+    width: 240px;
+    position: absolute;
```
`.tabs` should only be absolute if they have a `position: relative` parent. Right now, its parent is the `.pageMain` which is a bad practice. I suggest that you wrap tabs in a relative positioned div.

thi4go commented on src/App.js:
```diff
@@ -2,13 +2,17 @@ import React, { Suspense } from "react";
+        <ErrorsBoundary>
+          <Routes />
+        </ErrorsBoundary>
+        >
```
Hey mate. Right now, the `Results.jsx` component is where we throw the error from the server. If we put the `ErrorBoundary` on the `Page.jsx`, as it should be, it doesn't catch the error because the `Results.jsx` component doesn't render anything, not even `Page.jsx`. This will be fixed in the upcoming PR regarding the results page, and `ErrorBoundary` will be placed correctly. This is just temporary, so that it still catches errors once this PR goes in.

thi4go commented:
Hey @tiagoalvesdulce, I refactored the code considering what we discussed, and fixed all the review points. This is ready for another review :+1: since I was refactoring a lot of the component structure and css, I added breakpoints for mobile views and smaller screens, so this part can be reviewed also in this PR.

tiagoalvesdulce commented on src/components/Layout/Page.module.css:
```diff
@@ -0,0 +1,39 @@
+  margin-top: 6rem !important;
+  margin-left: 50px !important;
+  padding-bottom: 12rem;
+}
```
This isn't the best way to position this div. Consider using `flex` as I suggested and it should improve readability and reduce the dependency of specific breakpoints. This should be enough:

```
.pageMain {
    grid-row: auto !important;
    margin-top: 6rem !important;
    display: flex;
    flex-direction: column;
    align-items: center;
}

.mainContent {
    max-width: 540px;
}
```

thi4go replied:
Finally saw what you meant, mate. Fixed this.

tiagoalvesdulce commented on src/index.css:
```diff
@@ -60,32 +60,50 @@ code {
+  color: var(--color-gray-dark);
+  padding-top: 20px;
+  padding-right: 80px;
+  text-align: left;
```
This shouldn't be here. You can create custom components for this.

thi4go replied:
Fixed

tiagoalvesdulce commented on src/index.css:
```diff
@@ -60,32 +60,50 @@ code {
 
+/* Dropzone styles */
+
 .dropzone {
```
Why are the dropzone styles here?

thi4go replied:
It was already there. Will move them to the correct place

thi4go commented:
Addressed the remaining comments, @tiagoalvesdulce ! Should be good to go
